Mathlib/Data/Real/Sqrt.lean

```diff
@@ -124,6 +124,10 @@ theorem sqrt_div (x y : ℝ≥0) : sqrt (x / y) = sqrt x / sqrt y :=
 theorem continuous_sqrt : Continuous sqrt := sqrt.continuous
 #align nnreal.continuous_sqrt NNReal.continuous_sqrt
 
+@[simp] theorem sqrt_pos : 0 < sqrt x ↔ 0 < x := by simp [pos_iff_ne_zero]
```

Comment:
It looks to me like this theorem does not exist in mathlib3. I suppose the reason we need it here and not there is that we're adding a positivity extension for `NNReal`, and no such extension exists in mathlib3?

Reply:
Yes, that's right!